celery/utils/term.py

```diff
@@ -165,7 +165,11 @@ def __add__(self, other: object) -> str:
 
 
 def supports_images() -> bool:
-    return sys.stdin.isatty() and ITERM_PROFILE is not None
+
+    try:
```

Comment:
@Nusnus except this part was little bit changed from a suggestion in the report for the revert